cmd/bosun/web/static/js/incident.ts

```diff
@@ -0,0 +1,27 @@
+interface IIncidentScope extends ng.IScope {
+	error: string;
+	incident: any;
+	events: any;
+	actions: any;
+}
+
+bosunControllers.controller('IncidentCtrl', ['$scope', '$http', '$location', '$route', function($scope: IIncidentScope, $http: ng.IHttpService, $location: ng.ILocationService, $route: ng.route.IRouteService) {
+	var search = $location.search();
+	var id = search.id;
+	if (!id) {
+		$scope.error = "must supply incident id as query parameter"
+		return
+	}
+	$http.get('/api/incidents/events?id='+id)
+		.success((data) => {
+			$scope.incident = data.Incident;
+			if (moment($scope.incident.End).year() == 0) {
```

Comment:
I have three ideas for how to accomplish this and I hate all of them:

1. Create a new struct type just for web that has a string instead of a time.Time for End. Manually convert in those two calls.

2. Add an EndString or similar field to incident that we manually populate before serving. Serve that in json as `End`.

3. Create JSONTime struct as alias to time.Time that implements Marshaller interface: http://stackoverflow.com/a/23695774/121660 Use that in place of time.